pkg/agent/clientset.go

```diff
@@ -186,6 +186,9 @@ func (cs *ClientSet) sync() {
 				klog.V(4).InfoS("duplicate server", "serverID", dse.ServerID, "serverCount", cs.serverCount, "clientsCount", cs.ClientsCount())
 				if cs.serverCount != 0 && cs.ClientsCount() >= cs.serverCount {
 					duration = backoff.Step()
+				} else if cs.ClientsCount() < cs.serverCount {
```

Comment:
nit: in a given for loop iteration, let's call cs.ClientsCount() just once.

Reply:
Done!